cmd/record.go

```diff
@@ -49,25 +49,16 @@ func (t *Record) GetRecordConfig(path *string, proxyPort *uint32, appCmd *string
 	if err != nil {
 		return fmt.Errorf("failed to get the record config from config file due to error: %s", err)
 	}
-	if len(*path) == 0 {
-		*path = confRecord.Path
-	}
+
```

Comment:
This change is not needed. We have specifically did this because we wanted to give more preference to flag over config file. Like if we have mentioned -c flag in command line and mention appcmd in config file, Keploy should prefer cmd specified in flag over config file.